### PR TITLE
BSP - Add ModalAI Flight Core v2 initial support

### PR DESCRIPTION
**Pull request by modaltb**
- Addition of Flight Core v2 BSP - primary customer facing change from FCv1 is move to H7 processor
- We're early in the release cycle but trying to target 1.13 and want to get it into master to promote testing
- Simple flight test:
https://review.px4.io/plot_app?log=4b3626a6-3dd6-44d7-9471-63579f653f76

### CODE REVIEW

**Comment by dagar** [comment]
At the moment we need it here as well. https://github.com/PX4/PX4-Autopilot/blob/2feb096fb9fdaae46f19e162542bf66a9dacbaf7/.github/workflows/compile_nuttx.yml#L46-L47

File: boards/modalai/fc-v2/src/timer_config.cpp [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/****************************************************************************
+ *
+ *   Copyright (C) 2012 PX4 Development Team. All rights reserved.
```
Comment (dagar) [inline boards/modalai/fc-v2/src/timer_config.cpp:3]:
```suggestion
 *   Copyright (C) 2021 PX4 Development Team. All rights reserved.
```

Might as well set these.

**Comment by modaltb** [comment]
> At the moment we need it here as well.
> 
> https://github.com/PX4/PX4-Autopilot/blob/2feb096fb9fdaae46f19e162542bf66a9dacbaf7/.github/workflows/compile_nuttx.yml#L46-L47

thanks, updated

File: boards/modalai/fc-v2/src/timer_config.cpp (hunk 1 above)
Comment (modaltb) [reply boards/modalai/fc-v2/src/timer_config.cpp]: updated a few found in the fc-v2 board dir

**Comment by dagar** [comment]
Bootloader compile failure.

Screen Shot 2021-05-27 at 9 14 38 PM

**Comment by dagar** [comment]
Looks like the path changed slightly in https://github.com/PX4/PX4-Autopilot/pull/17619.

**Review by dagar — APPROVED** [review]

**Pull request merged**